articles/aks/use-kms-etcd-encryption.md

```diff
@@ -333,7 +333,13 @@ After configuring KMS, you can enable [diagnostic-settings for key vault to chec
 
 ## Disable KMS
 
-Use the following command to disable KMS on existing cluster.
+Prior disabling KMS you can use the following Azure CLI command to find out if KMS is enabled 
```

Comment:
To improve grammar and sentence structure, suggest rewording to -

Before disabling KMS, you can use the following Azure CLI command to verify if KMS is enabled.

Reply:
implemented